guides/rails_guides/generator.rb

```diff
@@ -17,13 +17,14 @@ module RailsGuides
   class Generator
     GUIDES_RE = /\.(?:erb|md)\z/
 
-    def initialize(edge:, version:, all:, only:, kindle:, language:)
+    def initialize(edge:, version:, all:, only:, kindle:, language:, rtl: false)
```

Comment:
Regarding variable naming, I would prefer to have `right_to_left` instead of `rtl` for better readability.